### PR TITLE
[Bot-Ai-Qna] Add FeedbackRecord

### PR DESCRIPTION
**Pull request by VictorGrycuk**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java [hunk 1]
```diff
@@ -2,3 +2,65 @@
...
+ */
+public class FeedbackRecod {
+    @JsonProperty("userId")
+    private String userId;
+
+    @JsonProperty("userQuestion")
+    private String userQuestion;
+
+    @JsonProperty("qnaId")
+    private String qnaId;
```
Comment (Batta32) [inline libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java:17]: This is `Integer`, not `String` (see [here](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.AI.QnA/Models/FeedbackRecord.cs#L32). You should change the getter/setter too.

Be careful with this, you are missing these kind of things.

**Review by Batta32 — CHANGES_REQUESTED** [review]
- Change the props to `with..` in the methods

File: libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java [hunk 2]
```diff
@@ -28,8 +28,8 @@ public String getUserId() {
      * Sets the feedback recod's user ID.
      * @param userId The user ID.
```
Comment (Batta32) [inline libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java:29]:
```suggestion
     * @param withUserId The user ID.
```

File: libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java [hunk 3]
```diff
@@ -44,23 +44,23 @@ public String getUserQuestion() {
      * Sets question asked by the user.
      * @param userQuestion The user question.
```
Comment (Batta32) [inline libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java:45]:
```suggestion
     * @param withUserQuestion The user question.
```

File: libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java [hunk 4]
```diff
@@ -44,23 +44,23 @@ public String getUserQuestion() {
...
      * @return The QnA ID.
      */
-    public String getQnaId() {
+    public Integer getQnaId() {
         return this.qnaId;
     }
 
     /**
      * Sets the QnA ID.
      * @param qnaId The QnA ID.
```
Comment (Batta32) [inline libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecord.java:61]:
```suggestion
     * @param withQnaId The QnA ID.
```

**Review by Batta32 — CHANGES_REQUESTED** [review]

**Review by Batta32 — APPROVED** [review]
LGTM

**Review by fran893 — APPROVED** [review]

**Pull request merged**